packages/react-native-tab-view/src/PanResponderAdapter.tsx

```diff
@@ -24,6 +25,7 @@ type Props<T extends Route> = PagerProps & {
   layout: Layout;
   onIndexChange: (index: number) => void;
   navigationState: NavigationState<T>;
+  onPageSelected?: NativeProps['onPageSelected'];
```

Comment:
This prop should be implemented in `PanResponderAdapter` so that using this API works on platforms other than Android & iOS, not only type